### ISSUE TITLE
Use proper knowl markup for SMF pages

### ISSUE THREAD

**Issue by davidfarmer** [LMFDB/lmfdb#883]
For example,
lmfdb/siegel_modular_forms/templates/includes/Sp4Z_basic.html
contains
<a class="knowl-title" knowl="group.sp2gr">full modular group</a>
which should be
{{KNOWL('group.sp2gr', title='full modular group')}}

But I don't think you can easily make that change, because the page is plain HTML, not jinja.

**Comment by AndrewVSutherland** [comment]
The page http://www.lmfdb.org/ModularForm/GSp/Q/Sp4Z/ is rendered by the template file

lmfdb/siegel_modular_forms/templates/ModularForm_GSp4_Q_collection.html

which includes lmfdb/siegel_modular_forms/templates/includes/Sp4Z_basic.html (it decides which file to include based on the particular family of spaces being displayed).  This does not preven knowls from appearing in the included files, in fact lmfdb/siegel_modular_forms/templates/includes/Sp4Z_basic.html contains a knowl for the full modular group that is displayed on the page above.

Can this issue be closed?  I'm not arguing that the current implementation is ideal or that everything that should be knowlified is, simply that the particular problem described in this issue does not appear to occur in the current implementation (which perhaps has changed since this issue was posted).

**Comment by davidfarmer** [comment]
The knowl is coded by hand-written html, not by using the appropriate template syntax.
Maybe that is not the most persuasive reason to use our templates, but use templates we should.

I marked this as 2.0 specifically so that the people who watch over these things can ignore it
and be comfortable with the fact that this issue will be open for another year or so.  Is there
a better way to indicate that status?

**Comment by AndrewVSutherland** [comment]
@davidfarmer Marking it 2.0 is fine, the only reason I was looking at it is because I just fixed a bunch of more urgent Siegel modular forms issues (all those marked as production problems and a few other obviously broken things in the code (e.g. using eval and sage_eval) and figured I would take a quick look at the list of outstanding issues to see if there were any that can be easily fixed at the same time before submitting the pull request.

I might suggest editing your original opening comment to make it clearer exactly what the problem is.  I'd also be interested to know what you think the "right" solution is.  I can see situations where one might very well want to include an html file into a template file (I don't think this applies to the *_basic.html files, but I think it does apply to the dimension_table.html file which I just changed to fix some error handling and formatting issues -- it would have been annoying to have to make the exact same change to every file that includes dimension_table.html).
